bigstore/bigstore.py

```diff
@@ -528,3 +541,17 @@ def init():
     g().config("filter.bigstore-compress.smudge", "git-bigstore filter-smudge")
 
     mkdir_p(object_directory(default_hash_function_name))
+
+
+def assert_initialized():
+    """
+    Check the make sure `git bigstore init` has been called.
+    If not then print an error and exit(1)
+    """
+    try:
+        if g().config('filter.bigstore.clean') == 'git-bigstore filter-clean':
+            return  # repo config looks good
+    except git.exc.GitCommandError:
+        pass
```

Comment:
In this case, are we not in a Git repo? If so, we might want to display a different error message.